ndcube/ndcube.py

```diff
@@ -1016,6 +1016,9 @@ def add(self, value, handle_mask=np.logical_and):
         # return the new NDCube instance
         return self._new_instance(**kwargs)
 
+    def add(self, operation, value, handle_mask = np.logical_and):
+        return self._operate_arithmetic(operation, value, handle_mask)
```

Comment:
We already know what the operation is here, i.e. `"add"`, so we don't need to expose this arg to the user.
```suggestion
    def add(self, value, handle_mask=np.logical_and):
        return self._operate_arithmetic("add", value, handle_mask)
```

Same below for `multiply`.